examples/porting/osal/lv_example_osal.c

```diff
@@ -51,7 +51,8 @@ void lv_example_osal(void)
         LV_LOG_ERROR("Error initializing thread sync");
     }
 
-    if(lv_thread_init(&increment_thread, LV_THREAD_PRIO_MID, increment_thread_entry, 2048, NULL) != LV_RESULT_OK) {
+    if(lv_thread_init(&increment_thread, "inc_th", LV_THREAD_PRIO_MID, increment_thread_entry, 2048,
```

Comment:
I would suggest a more expressive name, in a OS environment short names like that may collide with other threads, for example: `lv_inc_thread` or `lv_increment_th` tells more about the thread.